tests/auth_tests/test_models.py

```diff
@@ -368,3 +368,15 @@ def test_set_password(self):
     def test_check_password(self):
         with self.assertRaisesMessage(NotImplementedError, self.no_repr_msg):
             self.user.check_password('password')
+
+
+class GroupTests(TestCase):
+    def test_str(self):
+        g = Group.objects.create(name='Users')
```

Comment:
You could avoid the object creation and use `SimpleTestCase`.
```suggestion
        g = Group(name='Users')
```